woocommerce/wc-admin
# Update tooltip title and labels in standard graphs

Fixes the section _Standard Graph_ of #373.

**Screenshots**
Before:
![image](https://user-images.githubusercontent.com/3616980/45812278-9b9fe680-bccf-11e8-9a89-71ae1d2cec5a.png)

After:
![image](https://user-images.githubusercontent.com/3616980/45812195-672c2a80-bccf-11e8-979d-b655a12471e3.png)


**Steps to test**
- Go to the _Revenue_ page under _Analytics_.
- Verify the tooltip displays _Gross revenue_ as the title and the dates of each line as the labels.
- Check both, line chart and bar chart.
- Go to the _Dashboard_ page.
- Verify the tooltip hadn't changed in that chart: it stills displays the date as the tooltip title and the product key as the labels.
